engine/dlib/src/dlib/log.cpp

```diff
@@ -237,15 +237,15 @@ static void dmLogUpdateNetwork()
                 if (connections_full)
                 {
                     dmLogError("Too many log connections opened");
-                    const char* resp = "1 Too many log connections opened\n";
-                    SendAll(client_socket, resp, strlen(resp));
+                    const char resp[] = "1 Too many log connections opened\n";
```

Comment:
No need to change to array.

Reply:
Changing to array makes `sizeof` give the size of the array and not the pointer.